controller/src/test/java/org/jboss/as/controller/persistence/yaml/YamlConfigurationExtensionTest.java

```diff
@@ -116,12 +117,45 @@ public void registerAttributes(ManagementResourceRegistration resourceRegistrati
                 resourceRegistration.registerReadWriteAttribute(valueAtt, null, new ModelOnlyWriteAttributeHandler(valueAtt));
             }
         };
+        PropertiesAttributeDefinition propertiesAtt = new PropertiesAttributeDefinition.Builder("props", false).build();
+        SimpleResourceDefinition propertyResource = new SimpleResourceDefinition(new SimpleResourceDefinition.Parameters(PathElement.pathElement("property"), descriptionResolver)
+                .setAddHandler(new AbstractBoottimeAddStepHandler(propertiesAtt) {
+                })) {
+            @Override
+            public void registerAttributes(ManagementResourceRegistration resourceRegistration) {
+                super.registerAttributes(resourceRegistration);
+                resourceRegistration.registerReadWriteAttribute(propertiesAtt, null, new ModelOnlyWriteAttributeHandler(propertiesAtt));
+            }
+            @Override
+            public void registerOperations(ManagementResourceRegistration resourceRegistration) {
+                super.registerOperations(resourceRegistration);
+                resourceRegistration.registerOperationHandler(SimpleOperationDefinitionBuilder.of(WRITE_ATTRIBUTE_OPERATION, descriptionResolver).build(), new ModelOnlyWriteAttributeHandler(propertiesAtt));
+            }
+        };
+        /* ObjectTypeAttributeDefinition complexAtt = ObjectTypeAttributeDefinition.create("object",
+                  SimpleAttributeDefinitionBuilder.create("a", STRING, false)
+                        .setAllowExpression(true)
+                        .setValidator(new StringLengthValidator(0, true, true))
+                        .build(),
+                 SimpleAttributeDefinitionBuilder.create("b", ModelType.BOOLEAN).setAllowExpression(true).build())
+                  .build();
+         ObjectMapAttributeDefinition map = ObjectMapAttributeDefinition.create("map",complexAtt).build();
+        SimpleResourceDefinition mapResource = new SimpleResourceDefinition(new SimpleResourceDefinition.Parameters(PathElement.pathElement("map"), descriptionResolver)
+                .setAddHandler(new AbstractBoottimeAddStepHandler(complexAtt) {
+                })) {
+            @Override
+            public void registerAttributes(ManagementResourceRegistration resourceRegistration) {
+                super.registerAttributes(resourceRegistration);
+                resourceRegistration.registerReadWriteAttribute(complexAtt, null, new ModelOnlyWriteAttributeHandler(valueAtt));
+            }
+        };*/
```

Comment:
Changes will be in Git so I'm more inclined to remove it directly instead of leaving the code commented out; otherwise, if you want to see it commented out you can add an explanatory comment about why you keep it instead of removing it.

My feeling is these sorts of comments tend to get obsolete and nobody will know why it was left there without a proper explanation.